src/s4c_the_church_rap_battle.tw2

```diff
@@ -0,0 +1,343 @@
+
+::The Church - Rap battle
+Moon Mystic’s voice boomed over the PA. “Where my juggalos at tonight?! Where my fine juggalettes? All my down ass ninjas in the house go woop woop!”
+
+(click: "woop woop")[$dissolve[The crowd responded with a hearty “WOOP WOOP!”
+
+“Alright, alright, alright, we about to go into the final freestyle battle of the night. This is what all you crazy dedicated ninjas out there have been sitting at the edge of your seat begging me for. This is what the bloodthirsty motherfuckers out there desire. This is...”
+
+Moon Mystic paused for maximum drama.]]
+
+(click: "paused")[$dissolve[“A MOUNTAIN FIGHT!”
+
+As the words left his lips, the whole crowd stood up and collectively lost their mind. The guy at the sound booth mixed a bunch of record scratches and crazy airhorn noises together.
+
+“Now, it’s time for the combatants. Defending the title, blocking the path, it’s your champion. The dirty diggler, the mouth of a thousand corpses, it’s Smooth Lips Steve!”]]
+
+(click: "Smooth Lips Steve")[$dissolve[Steve jumped out in full face paint, looking ready for a pick up game of basketball. He swung his arms down from the stage amiably, [[high-fiving his homies|The Church - Rap Battle player intro]].]]
+
+::The Church - Rap Battle player intro
+Moon Mystic’s voice boomed again, “And now, the fighter. Going for the mountain after an unprecedented year-long absence from the stage. She’s the little ‘lette, the fiercest female on the mic, it’s [$stageName]<clickName|!!
+
+(click: ?clickName)[$dissolve[The crowd was totally silent as I walked out on stage. I had no face paint. I was just wearing my torn-ass jeans and a beat up Milenko shirt. I probably still smelled like the port-a-potty and looked like I’d crawled out of tent city after a weekend bender. But my face was absolute granite. I didn’t move a muscle, wouldn’t give an inch.
+
+As I moved to pick my mic up from where it was coiled on the stage floor, some stupid-ass juggalo in the crowd called out, “Yo, I ain’t ever seen this bitch. She ain’t got it!” Steve popped a mile-wide grin and there was some laughter in the crowd.]]
+
+(click: "laughter in the crowd")[$dissolve[I quickly snatched up the mic and blasted back, “Yeah ninja, that ain’t a surprise. Looks to me like a lot of ‘lettes try to avoid you all together.” An immediate “OOOOOOH” from the crowd was my delicious reward. Steve’s grin dropped and he picked his mic up as well. [[Time to do some fuckin' raps|The Church - Rap battle - First verses]].]]
+
+::The Church - Rap battle - First verses
+Steve and I circled each other warily. Moon Mystic piped up, “Alright $stageName, as the challenger, you pick who goes first.”
+
+>>["I’ll go first. I’m sick of looking at this dude already."]<playerFirst|
+>>["I’ll go second. This goofy motherfucker needs all the help he can get."]<steveFirst|
+>>[[“I didn’t even answer, just started into a blistering rap about dogs.”|The Church - Rap battle - First verses dog rap]]
```

Comment:
This option shouldn't be in quotes, right?

Reply:
Also, I'd swap the comma for an em-dash

Reply:
whooooops